### PR TITLE
Allow to configure and disable the incubator repo

### PR DESCRIPTION
**Pull request by pvorb**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -20,11 +20,11 @@ Add the following to your `pom.xml`
     <plugin>
       <groupId>com.deviceinsight.helm</groupId>
       <artifactId>helm-maven-plugin</artifactId>
-      <version>2.1.0</version>
+      <version>2.7.2</version>
```
Comment (mwalser) [inline README.adoc:23]: Isn't `1.7.1` the latest version?

File: src/main/kotlin/com/deviceinsight/helm/PackageMojo.kt [hunk 2]
```diff
@@ -29,7 +29,7 @@ import java.io.File
 class PackageMojo : AbstractHelmMojo() {
 
 	companion object {
-		private val PLACEHOLDER_REGEX = Regex("""\$\{(.*?)\}""")
+		private val PLACEHOLDER_REGEX = Regex("""\$\{(.*?)}""")
```
Comment (mwalser) [inline src/main/kotlin/com/deviceinsight/helm/PackageMojo.kt:32]: I guess this was a unnecessarily escape?

File: README.adoc (hunk 1 above)
Comment (mwalser) [reply README.adoc:23]: I mean `2.7.1`

Comment (pvorb) [reply README.adoc:23]: oops

File: src/main/kotlin/com/deviceinsight/helm/PackageMojo.kt (hunk 2 above)
Comment (pvorb) [reply src/main/kotlin/com/deviceinsight/helm/PackageMojo.kt:32]: Exactly. Wouldn't have found it myself but IntelliJ was so kind to give me a hint. :wink:

File: README.adoc (hunk 1 above)
Comment (pvorb) [reply README.adoc]: fixed

**Pull request merged**